exchanges/lbank/lbank.go

```diff
@@ -10,11 +10,13 @@ import (
 	"encoding/pem"
 	"errors"
 	"fmt"
+	"github.com/thrasher-corp/gocryptotrader/currency"
```

Comment:
this line can be moved to [here](https://github.com/thrasher-corp/gocryptotrader/pull/414/files#diff-608a64c89ed4c421f10d496cd562f9ceR19)